File: workspace/3_EmployedWorkers/src/employedworker/Student.java
package employedworker;

/**
 * A Student has a unique student number
 */
public class Student {
	
	private long uNumber;
	
	public Student(String name) {	
		this.uNumber = name.hashCode()+System.currentTimeMillis();
	}
		
	public long getUNumber() {
		return uNumber;
	}
}

File: solution/3_EmployedWorkers/3_EmployedWorkers/src/employedworker/Student.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace src.employedworker
{
    /// <summary>
    /// Een Student heeft een uniek studentnummer
    /// </summary>
    public class Student
    {
        private long uNumber;

        public Student(string name)
        {
            this.uNumber = name.GetHashCode() + new DateTime().Ticks;
        }

        public long getUNumber()
        {
            return uNumber;
        }
    }
}
